docs/partials/coming-soon-centered.md

```diff
@@ -0,0 +1 @@
+# <center>🚧 Work in Progress... 🚧</center>
```

Comment:
This looks MASSIVE when rendered as an H1. I'd rather see this as normal text or maybe an [Admonition](https://docusaurus.io/docs/markdown-features/admonitions)

Reply:
Will try admonition

Reply:
![image](https://user-images.githubusercontent.com/24318071/141661802-cd8dc3a4-f0c0-4762-82f0-b87f24ef52cb.png)